### PR TITLE
Font size adjustment

### LINKED ISSUE #10: allow window / font-size adjustment.
I've got a large, high-resolution screen and need the labels to be larger. Thanks!

I've waiting for years for a tool like this to work on Wayland. Thanks so much!

### PR DESCRIPTION
**Pull request by edzdez**
Addresses Issue #10

### CODE REVIEW

File: src/cli.rs [hunk 1]
```diff
@@ -26,65 +26,72 @@ pub struct Args {
     #[arg(long)]
     pub label_text_color: Option<String>,
 
-    // set the font family
+    /// set the font family
```
Comment (markstos) [inline src/cli.rs:29]: Thanks for adding font-size support! Did you mean to add the extra slashes here?

Comment (edzdez) [reply src/cli.rs:29]: Yep! Putting the extra slashes there makes the comment a doc-comment, which `clap` helpfully parses into the cli `--help` message.

**Comment by markstos** [comment]
I played with a version of this locally and found that simply setting the font-size to be larger does indeed make the whole window larger.

File: src/cli.rs (hunk 1 above)
Comment (markstos) [reply src/cli.rs:29]: Cool! I've used other CLI frameworks which can generate help from comments and it's a nice feature.

**Pull request merged**